webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,20 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/<user_api_key>/get_pending_jobs", methods=["GET"])
+def get_pending_jobs(user_api_key):
+    """ Return the pending jobs for a user.
+
+    Retrieve the Job Ids for the pending jobs for the user whose
+    user id is given as an argument and return that.
+    """
+    final_result = {}
+    try:
+        pending_jobs = db.dataset_eval.get_user_pending_jobs(user_api_key)
+        for i in range(len(pending_jobs)):
+            final_result[i+1] = str(pending_jobs[i][0])
+    except:
+        e = sys.exc_info()[0]
```

Comment:
You can decide how you want to check this in the database query. If you are logged in then the user will exist. If you use the auth system then you don't need to check if the api key is valid. You should decide what you want the behaviour to be if the user has no datasets to evaluate.